### DISCUSSION TITLE
Failure to fetch go-fsnotify?

### DISCUSSION THREAD

**Discussion by tmdarwen** [OE4T/tegra-demo-distro#185] [Q&A] [answered]
I've built the tegra-demo-distro image before without problems.  Now, when doing it from scratch I'm getting the following warning and error:
```
WARNING: go-fsnotify-1.2.11+gitAUTOINC+836bfd95fe-r0 do_fetch: Failed to fetch URL git://github.com/fsnotify/fsnotify.git;branch=master;protocol=https, attempting MIRRORS if available
ERROR: go-fsnotify-1.2.11+gitAUTOINC+836bfd95fe-r0 do_fetch: Fetcher failure: Unable to find revision 836bfd95fecc0f1511dd66bdbf2b5b61ab8b00b6 in branch master even from upstream
```

Looking at the log file, I see it fails when attempting to fetch it from this location:
```
http://downloads.yoctoproject.org/mirror/sources/git2_github.com.fsnotify.fsnotify.git.tar.gz
```
It fails with a 404 error:
```
Resolving downloads.yoctoproject.org (downloads.yoctoproject.org)... 198.145.29.63
Connecting to downloads.yoctoproject.org (downloads.yoctoproject.org)|198.145.29.63|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-01-21 22:31:59 ERROR 404: Not Found.
```

When trying to go to this location with a browser I get a 404 error.  Additionally, I can go to http://downloads.yoctoproject.org/mirror/sources/ and see that the git2_github.com.fsnotify.fsnotify.git.tar.gz is not listed.

Any idea how to deal with this?

**Comment by ichergui** [comment, marked as answer]
Hello @tmdarwen 

You need to update `meta-virtualization` otherwise please update `tegra-demo-distro` by using the following command line:

```
$ cd /path/to/tegra-demo-distro
$ git submodule update --init --recursive
```

Then, you trig your build, it should be fine
